server/team/team.contoller.js

```diff
@@ -2,20 +2,109 @@ const debug = require('debug');
 
 const Team = require('./team.model');
 const User = require('../user/user.model');
+const launchChromeAndRunLighthouse = require('../lighthouse/lighthouse');
 
 const log = debug('fcc:team:controller');
 
 function create(req, res) {
   const { body: team } = req;
-  log(team);
-  return Team.create(team)
-    .then((newTeam) => {
-      if (!newTeam) {
-        return res.status(500).json({ acknowladged: false });
-      }
-      res.status(200).json({ acknowladged: true });
-      return newTeam._id;
-    })
-    .then(teamId => User.update({ _id: req.user._id }, { teamId }));
+  const collaborators = team.collaborators.split(',').map(str => str.trim())
+    .filter(Boolean);
+  team.collaborators = collaborators;
+  const newTeam = new Team(team);
+  return newTeam.save()
+  .then(() => {
+    if (!newTeam) {
+      return res.status(500).json({ acknowladged: false });
+    }
+    return User.findOneAndUpdate(
+      { _id: req.user._id },
+      { $set: { teamId: newTeam._id } }
+    )
+    .then((user) => {
+      res.redirect(`/api/teams/${user.teamId}`);
```

Comment:
Sorry, this should probably be res.redirect('/teams') ... and same with update()